homeassistant/components/mqtt/cover.py

```diff
@@ -370,8 +388,23 @@ def name(self):
 
     @property
     def is_closed(self):
-        """Return if the cover is closed."""
-        return self._state
+        """Return true if the cover is closed or None if the status is unknown."""
+        return None if self._state == STATE_UNKNOWN else self._state == STATE_CLOSED
+
+    @property
+    def is_open(self):
```

Comment:
Where is this used?

Reply:
Good catch, I assumed it was defined in `CoverDevice` since `is_closed` is, but it appears `is_closed` suffices :)

Reply:
Good catch, I assumed it was defined in `CoverDevice` since `is_closed` is, but it appears `is_closed` suffices :)